selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Editor.java

```diff
@@ -0,0 +1,1914 @@
+        actionsFactory.createAction(seleniumWebDriver).sendKeys(text).perform();
+        loader.waitOnClosed();
+    }
+
+    /**
+     * type text into orion editor pause for saving on server side is not set
+     *
+     * @param text
+     *         text to type
+     */
+    public void typeTextIntoEditorWithoutDelayForSaving(String text) {
+        actionsFactory.createAction(seleniumWebDriver).sendKeys(text).perform();
+    }
+
+    /**
+     * type text into specific line editor. Cursor will be activated id the end of the line. Line must
+     * not be empty. Not recommended to use
+     *
+     * @param text
+     *         text to type
+     */
+    public void typeTextIntoEditor(String text, int line) {
+        setCursorToLine(line);
+        typeTextIntoEditor(text);
+    }
+
+    /** returns focus in the end of current line (in active tab) */
+    // TODO in some cases (for example if we do step into in debug mode and opens editor after that,
+    // focus will be lost). But this problem should be fixed. After the we can remove this method
+    public void returnFocusInCurrentLine() {
+        List<WebElement> lines =
+                seleniumWebDriver.findElements(By.xpath(Locators.ACTIVE_LINE_HIGHLIGHT));
+        Collections.sort(
+                lines, (o1, o2) -> Integer.compare(o1.getLocation().getX(), o2.getLocation().getX()));
+        lines = lines.subList(0, lines.size() / 2); // filter lines from preview
+        for (WebElement line : lines) {
+            if (line.isDisplayed()) {
+                try {
+                    int lineWidth = line.getSize().getWidth();
+                    Actions action = actionsFactory.createAction(seleniumWebDriver);
+                    action.moveToElement(line, lineWidth - 1, 0).click().perform();
+                } catch (Exception ignore) {
+                }
+            }
+        }
+    }
+
+    /**
+     * set cursor in specified line
+     *
+     * @param positionLine
+     *         is the specified number line
+     */
+    public void setCursorToLine(int positionLine) {
+        loader.waitOnClosed();
+        actionsFactory
+                .createAction(seleniumWebDriver)
+                .sendKeys(Keys.chord(Keys.CONTROL, "l"))
+                .perform();
+        askForValueDialog.waitFormToOpen();
+        loader.waitOnClosed();
+        askForValueDialog.typeAndWaitText(String.valueOf(positionLine));
+        loader.waitOnClosed();
+        askForValueDialog.clickOkBtn();
+        askForValueDialog.waitFormToClose();
+        waitActiveEditor();
+        waitExpectedNumberOfActiveLine(positionLine);
+    }
+
+    /**
+     * set cursor in specified position for current visible editor
+     *
+     * @param positionLine
+     *         is the specified number line
+     * @param positionChar
+     *         is the specified number char
+     */
+    public void setCursorToDefinedLineAndCharForCurrentVisibleEditor(int positionLine, int positionChar) {
```

Comment:
Method name **setCursorToDefinedLineAndCharForCurrentVisibleEditor()** looks to long, IMHO.
Can we use here **goToPositionInActiveTab()**?